handler.go

```diff
@@ -39,8 +41,9 @@ type pathConfig struct {
 
 func newHandler(config []byte) (*handler, error) {
 	var parsed struct {
-		Host  string `yaml:"host,omitempty"`
-		Paths map[string]struct {
+		Host     string `yaml:"host,omitempty"`
+		CacheAge *int64 `yaml:"cache_age,omitempty"`
```

Comment:
Should we call it cache_max_age or just cache_secs or something in the lines of that? People already are familiar with max age and can guess the unit.

Reply:
Done.